src/coredump/coredumpctl.c

```diff
@@ -467,6 +483,19 @@ static void analyze_coredump_file(
         *ret_size = UINT64_MAX;
 }
 
+static int get_absolute_filename(sd_journal *j, char **filename) {
```

Comment:
"get" is a bit of a misnomer I think. maybe call this "make", i.e. "make_filename_absolute()"